terraform/testcases/datadog_exporter_trace_mock/otconfig.tpl

```diff
@@ -0,0 +1,27 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:${grpc_port}
+
+processors:
+  batch:
+    timeout: 10s
+
+exporters:
+  logging:
+    loglevel: debug
+  datadog:
+    api:
+      key: testapikey
+    metrics:
```

Comment:
could you remove the metric block here since the test case is for trace?

Reply:
I should have made a comment about this, my bad: this is necessary because we actually send some stats about traces to that endpoint when running the trace exporter; if we leave it unset we would be hitting Datadog's backend on each test

Reply:
Make sense then, thx for explaining.